kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/DefaultConfigurationFactory.java

```diff
@@ -41,6 +41,8 @@ protected static void configureProtocolHandlers(Map<String, String> conf) {
     public C create(ArquillianDescriptor arquillian) {
         Map<String, String> config = arquillian.extension(KUBERNETES_EXTENSION_NAME).getExtensionProperties();
         configureProtocolHandlers(config);
-        return (C) DefaultConfiguration.fromMap(config);
+        final DefaultConfiguration configuration = DefaultConfiguration.fromMap(config);
+        System.out.println(configuration);
```

Comment:
Two things to check here:

1) Are we using `sysout` in Docker part as well? Now I don't remember exactly if we changed to `logger` or not.
2) Have you checked if at this stage in case of using random namespace (automatically generated) is already set or not? I think (but I have not tried) that in that specific case the namespace will not be shown. So I suggest moving this printing in a stage where it has been already calculated and set. (For example `BeforeSuite` event). But something to check before doing the change.

Reply:
> 1. Are we using `sysout` in Docker part as well? Now I don't remember exactly if we changed to `logger` or not.

It is still using `sysout` - https://github.com/arquillian/arquillian-cube/blob/4bed6df837f24d06813b6047cd223c81f508f6a6/docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfigurator.java#L75 
Should this be changed to `logger`? 

> 2. Have you checked if at this stage in case of using random namespace (automatically generated) is already set or not? I think (but I have not tried) that in that specific case the namespace will not be shown. So I suggest moving this printing in a stage where it has been already calculated and set. (For example BeforeSuite event). But something to check before doing the change.

Yes, I checked. The namespace is set in the case when it is automatically generated.

![screenshot from 2018-02-26 16-07-44](https://user-images.githubusercontent.com/8680543/36713299-e3ea2bd0-1bb2-11e8-8f1f-ecc3a1d67dc2.png)